m/configDataPreprocessingSeaexplorer.m

```diff
@@ -0,0 +1,129 @@
+function preprocessing_options = configDataPreprocessingSeaexplorer()
+%CONFIGDATAPREPROCESSINGSEAEXPLORER  Configure Seaexplorer glider data preprocessing.
+%
+%  Syntax:
+%    PREPROCESSING_OPTIONS = CONFIGDATAPREPROCESSINGSEAEXPLORER()
+%
+%  Description:
+%    PREPROCESSING_OPTIONS = CONFIGDATAPREPROCESSINGSEAEXPLORER() should return 
+%    a struct setting the options for Seaexplorer glider data preprocessing 
+%    as needed by the function PREPROCESSGLIDERDATA.
+%
+%  Examples:
+%    preprocessing_options = configDataPreprocessingSeaexplorer()
+%
+%  See also:
+%    PREPROCESSGLIDERDATA
+%
+%  Authors:
+%    Frederic Cyr  <Frederic.Cyr@mio.osupytheas.fr>
+%    Joan Pau Beltran  <joanpau.beltran@socib.cat>
+
+%  Copyright (C) 2013-2015
+%  ICTS SOCIB - Servei d'observacio i prediccio costaner de les Illes Balears
+%  <http://www.socib.es>
+%
+%  This program is free software: you can redistribute it and/or modify
+%  it under the terms of the GNU General Public License as published by
+%  the Free Software Foundation, either version 3 of the License, or
+%  (at your option) any later version.
+%
+%  This program is distributed in the hope that it will be useful,
+%  but WITHOUT ANY WARRANTY; without even the implied warranty of
+%  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+%  GNU General Public License for more details.
+%
+%  You should have received a copy of the GNU General Public License
+%  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+  error(nargchk(0, 0, nargin, 'struct'));
+  
+  preprocessing_options = struct();
+
+  %% Navigation
+  preprocessing_options.time_list(1).time = 'Posixtime_sci';
+  preprocessing_options.position_list(1).longitude = 'NAV_LONGITUDE';
+  preprocessing_options.position_list(1).latitude  = 'NAV_LATITUDE';
+  preprocessing_options.position_list(1).conversion = @nmea2deg;
+  
+  %preprocessing_options.depth_list(1).depth = 'NAV_DEPTH';
+  preprocessing_options.depth_list(1).depth = 'GPCTD_PRESSURE';
+  preprocessing_options.depth_list(2).depth = 'SBD_PRESSURE';
+  %preprocessing_options.depth_list(3).depth = 'NAV_DEPTH';
+
```

Comment:
Is there any reason to not use the navigation depth `Depth` here?
Note that, in addition to this depth, you can also enable the depth derivation from CTD pressure,
(which derives the depth using the pressure and the latitude).
You will end up with two variables: `depth` and `depth_ctd`.
They might or might not coincide, depending on how the glider works and the sampling configuration options.
Does the SeaExplorer have a dedicated pressure sensor for navigation or uses the one from the CTD?
Can/Do the navigation depth have a different sampling configuration than the CTD pressure?
Sometimes it might useful to have both of them, for example if the CTD sampling is disabled
while other sensors are enabled.

Of course this is just the default configuration, users can adjust it to their need in their runtime environment.